x-pack/platform/plugins/shared/entity_manager/server/plugin.ts

```diff
@@ -197,6 +201,9 @@ export class EntityManagerServerPlugin
       getScopedClient: async ({ request }: { request: KibanaRequest }) => {
         return this.getScopedClient({ request, coreStart: core });
       },
+      v2: {
+        instanceAsFilter,
+      },
```

Comment:
any pro exposing the function like this instead of a static import ?

Reply:
It's minor but it makes it more clear that you need to depend on our _bundle_ to use this function.

If we just do a static export/import, people might miss to update their required bundles? 
I'm not sure if there are CI rules that check for this? 
I might be overthinking it :)

Should I just change it to a static import?

Reply:
are required bundles a thing for server side code ?

Reply:
I don't know, we can change this later if need be